cybersixgill_actionable_alerts/datadog_checks/cybersixgill_actionable_alerts/check.py

```diff
@@ -0,0 +1,143 @@
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+Channel_Id = "d5cd46c205c20c87006b55a18b106428"
+
+
+class CybersixgillActionableAlertsCheck(AgentCheck):
+    SERVICE_CHECK_CONNECT_NAME = 'cybersixgill.can_connect'
+    SERVICE_CHECK_HEALTH_NAME = 'cybersixgill.health'
+
+    def __init__(self, name, init_config, instances):
+        super(CybersixgillActionableAlertsCheck, self).__init__(name, init_config, instances)
+
+    def check(self, instance):
+        cl_id = self.instance.get('client_id')
+        cl_secret = self.instance.get('client_secret')
+        if not cl_id and cl_secret:
+            raise ConfigurationError
+        alerts_limit = self.instance.get('alerts_limit')
+        threat_level = self.instance.get('threat_level')
+        threat_type = self.instance.get('threat_type')
+        if not threat_level:
+            threat_level = None
+        threat_type_list = None
+        if threat_type:
+            threat_type_list = threat_type.split(", ")
+        try:
+            file_dir = os.path.dirname(__file__)
+            abs_file_path = os.path.join(file_dir, "date_threshold.txt")
+            if os.path.exists(abs_file_path):
+                file_data = open(abs_file_path, 'r')
+                req_date = file_data.read()
+                date_conv = json.loads(req_date)
+                time_stamp_str = date_conv["from_date"]
+                if time_stamp_str is not None:
+                    from_datetime = datetime.strptime(time_stamp_str, "%Y-%m-%d %H:%M:%S")
+                else:
+                    from_datetime = datetime.now()
+                    str_from_date = from_datetime.strftime("%Y-%m-%d %H:%M:%S")
+                    str_from_date = datetime.strptime(str_from_date, "%Y-%m-%d %H:%M:%S")
+                    from_datetime = str_from_date - timedelta(days=90)
+            else:
+                from_datetime = datetime.now()
+                str_from_date = from_datetime.strftime("%Y-%m-%d %H:%M:%S")
+                str_from_date = datetime.strptime(str_from_date, "%Y-%m-%d %H:%M:%S")
+                from_datetime = str_from_date - timedelta(days=90)
+            to_datetime = datetime.now()
+            str_today = to_datetime.strftime("%Y-%m-%d %H:%M:%S")
+            str_today = datetime.strptime(str_today, "%Y-%m-%d %H:%M:%S")
+            to_datetime = str_today
+            sixgill_client = SixgillActionableAlertClient(
+                client_id=cl_id, client_secret=cl_secret, channel_id=Channel_Id, verify=True
+            )
+            alerts = sixgill_client.get_actionable_alerts_bulk(
+                limit=alerts_limit,
+                from_date=from_datetime,
+                to_date=to_datetime,
+                threat_type=threat_type_list,
+                threat_level=threat_level,
+            )
+            for al in alerts:
+                alert_id = al.get("id")
+                portal_url = f"https://portal.cybersixgill.com/#/?actionable_alert={alert_id}"
```

Comment:
From your `hatch.toml`, looks like you support Python 2. F-strings are not supported on Python 2. Instead, can you use:

```
"https://portal.cybersixgill.com/#/?actionable_alert=%s" % alert_id
```

Reply:
@emarsha94 I have made that change and committed the code.